### PR TITLE
[Update] Add TiSpark telemetry 

### PR DESCRIPTION
**Pull request by Daemonxiao**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

- TiSpark has telemetry in master branch and will have release the telemetry function in the next version. So we should add TiSpark telemetry instructions in telemtry.md.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): [TiSpark Telemetry](https://github.com/pingcap/tispark/blob/master/docs/telemetry.md)

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: telemetry.md [hunk 1]
```diff
@@ -60,6 +60,22 @@ ADMIN SHOW TELEMETRY;
 TIUP_CLUSTER_DEBUG=enable tiup cluster list
 ```
 
+### TiSpark
+
+当 TiSpark 遥测功能开启时，Spark 在使用 TiSpark 时会发送会将使用情况信息分享给 PingCAP，包括（但不限于）：
```
Comment (xuanyu66) [inline telemetry.md:65]: Please list the detail message, not `包括（但不限于）`

**Review by xuanyu66 — CHANGES_REQUESTED** [review]

File: telemetry.md (hunk 1 above)
Comment (Daemonxiao) [reply telemetry.md:65]: Hmmm, but I should follow the format above, such as TiDB, and TiUP. It seems a simple summary is better here.

Comment (xuanyu66) [reply telemetry.md:65]: LGTM

**Review by xuanyu66 — APPROVED** [review]

File: telemetry.md [hunk 2]
```diff
@@ -60,6 +60,22 @@ ADMIN SHOW TELEMETRY;
...
+
+- 随机生成的遥测标示符
+- TiSpark 的部分配置信息，如读取引擎、是否开启流式读取等
+- 用户集群部署情况，包括 TiSpark 所在节点的机器硬件信息、操作系统信息和组件版本号等
+
+当 TiSpark 被使用时，可以通过查看 Spark 日志来了解 TiSpark 收集的使用情况信息内容，需要把 Spark 日志级别调至 INFO 或更低，例如：
```
[the remark this answers is not in the document]
Comment (Daemonxiao) [reply telemetry.md]: @shichun-0415 Thanks for your review. I have commit suggestion.

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**